<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sccGuiClass</class>
 <widget class="QMainWindow" name="sccGuiClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sccGui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="bmcToolBar">
   <property name="windowTitle">
    <string>BMC Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolsToolBar">
   <property name="windowTitle">
    <string>Tools Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="widgetToolBar">
   <property name="windowTitle">
    <string>Widget Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionClearLogWindow"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuGuiSettings">
     <property name="title">
      <string>GUI settings</string>
     </property>
     <addaction name="actionSplashscreenEnabled"/>
     <addaction name="actionSaveGeometry"/>
     <addaction name="actionRestoreGeometry"/>
    </widget>
    <widget class="QMenu" name="menuDebugSettings">
     <property name="title">
      <string>Debug settings</string>
     </property>
     <addaction name="actionSetPacketTraceFile"/>
     <addaction name="actionSetPacketTraceMode"/>
     <addaction name="actionEnableSoftwareRamAndUart"/>
    </widget>
    <widget class="QMenu" name="menuLinuxBootSettings">
     <property name="title">
      <string>Linux boot settings</string>
     </property>
     <addaction name="actionLinuxEnableL2"/>
     <addaction name="actionLinuxClearMPBs"/>
     <addaction name="actionLinuxSelectImage"/>
     <addaction name="actionLinuxRestoreImage"/>
     <addaction name="actionLinuxBootAll"/>
     <addaction name="actionSetLinuxGeometry"/>
     <addaction name="actionPopshmSettings"/>
     <addaction name="actionCmdLineArgs"/>
    </widget>
    <addaction name="menuGuiSettings"/>
    <addaction name="menuDebugSettings"/>
    <addaction name="menuLinuxBootSettings"/>
   </widget>
   <widget class="QMenu" name="menuWidgets">
    <property name="title">
     <string>Widgets</string>
    </property>
    <addaction name="actionFlitWidget"/>
    <addaction name="actionPacketWidget"/>
    <addaction name="actionReadMemWidget"/>
    <addaction name="actionPerfmeterWidget"/>
    <addaction name="actionStartkScreen"/>
   </widget>
   <widget class="QMenu" name="menuBMC">
    <property name="title">
     <string>BMC</string>
    </property>
    <addaction name="actionInitializePlatform"/>
    <addaction name="actionTrainSystemInterface"/>
    <addaction name="actionExecuteRccFile"/>
    <addaction name="actionGetBoardStatus"/>
    <addaction name="actionResetPlatform"/>
    <addaction name="actionCustomBmcCmd"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionWischer"/>
    <addaction name="actionBootLinux"/>
    <addaction name="actionTestMemory"/>
    <addaction name="actionPreloadObject"/>
    <addaction name="actionPreloadLut"/>
    <addaction name="actionClearMPBs"/>
    <addaction name="actionChangeL2Enables"/>
    <addaction name="actionChangeResets"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBMC"/>
   <addaction name="menuTools"/>
   <addaction name="menuWidgets"/>
   <addaction name="menuSettings"/>
   <addaction name="separator"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/exit.png</normaloff>:/resources/menuIcons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Close sccGui...</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClearLogWindow">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/garbageEmpty.png</normaloff>:/resources/menuIcons/garbageEmpty.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear log window</string>
   </property>
   <property name="iconText">
    <string>Clear log window</string>
   </property>
   <property name="toolTip">
    <string>Clear log window</string>
   </property>
  </action>
  <action name="actionSplashscreenEnabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Splashscreen enabled</string>
   </property>
  </action>
  <action name="actionSaveGeometry">
   <property name="text">
    <string>Save current window geometry</string>
   </property>
   <property name="toolTip">
    <string>Saves current window geometry for next start of the GUI...</string>
   </property>
  </action>
  <action name="actionRestoreGeometry">
   <property name="text">
    <string>Restore default window geometry</string>
   </property>
   <property name="toolTip">
    <string>Restores the default window geometry (centered, 640x480)...</string>
   </property>
  </action>
  <action name="actionFlitWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/flitWidget.png</normaloff>:/resources/menuIcons/flitWidget.png</iconset>
   </property>
   <property name="text">
    <string>Flit widget (NC reads and writes)</string>
   </property>
   <property name="toolTip">
    <string>Open flit widget...</string>
   </property>
  </action>
  <action name="actionPacketWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/packetWidget.png</normaloff>:/resources/menuIcons/packetWidget.png</iconset>
   </property>
   <property name="text">
    <string>Generic packet transfer widget</string>
   </property>
   <property name="toolTip">
    <string>Open packet widget...</string>
   </property>
  </action>
  <action name="actionReadMemWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/ram.png</normaloff>:/resources/menuIcons/ram.png</iconset>
   </property>
   <property name="text">
    <string>Memory reader widget</string>
   </property>
   <property name="toolTip">
    <string>Open memory reader widget...</string>
   </property>
  </action>
  <action name="actionPerfmeterWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/perfmeter.png</normaloff>:/resources/menuIcons/perfmeter.png</iconset>
   </property>
   <property name="text">
    <string>SCC performance meter</string>
   </property>
   <property name="toolTip">
    <string>Open performance meter widget...</string>
   </property>
  </action>
  <action name="actionSetPacketTraceFile">
   <property name="text">
    <string>Set packet trace file</string>
   </property>
  </action>
  <action name="actionSetPacketTraceMode">
   <property name="text">
    <string>Set packet trace mode</string>
   </property>
  </action>
  <action name="actionSelectActiveCores">
   <property name="text">
    <string>Select active cores</string>
   </property>
  </action>
  <action name="actionBootLinux">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/linuxBlue.png</normaloff>:/resources/menuIcons/linuxBlue.png</iconset>
   </property>
   <property name="text">
    <string>Boot Linux...</string>
   </property>
   <property name="toolTip">
    <string>Boot Linux</string>
   </property>
   <property name="statusTip">
    <string>Linux not booted! Click icon to start booting...</string>
   </property>
  </action>
  <action name="actionLinuxEnableL2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable L2</string>
   </property>
  </action>
  <action name="actionLinuxSelectImage">
   <property name="text">
    <string>Choose custom Linux image (file dialog)</string>
   </property>
  </action>
  <action name="actionLinuxBootAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always boot all 48 cores</string>
   </property>
  </action>
  <action name="actionPreloadObject">
   <property name="text">
    <string>Preload object file</string>
   </property>
  </action>
  <action name="actionChangeResets">
   <property name="text">
    <string>Change selected reset(s)</string>
   </property>
  </action>
  <action name="actionWischer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/wischer.png</normaloff>:/resources/menuIcons/wischer.png</iconset>
   </property>
   <property name="text">
    <string>Software SCC reset</string>
   </property>
   <property name="toolTip">
    <string>Resets all cores and Tile registers...</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionToggleSelectedResets">
   <property name="text">
    <string>Toggle selected resets</string>
   </property>
  </action>
  <action name="actionLinuxClearMPBs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clear MPBs before booting</string>
   </property>
  </action>
  <action name="actionPreloadLut">
   <property name="text">
    <string>Preload LUT file</string>
   </property>
  </action>
  <action name="actionChangeL2Enables">
   <property name="text">
    <string>Change L2 enable(s)</string>
   </property>
  </action>
  <action name="actionClearMPBs">
   <property name="text">
    <string>Clear MPB(s)</string>
   </property>
  </action>
  <action name="actionGetBoardStatus">
   <property name="text">
    <string>Get board status</string>
   </property>
  </action>
  <action name="actionInitializePlatform">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/radiation.png</normaloff>:/resources/menuIcons/radiation.png</iconset>
   </property>
   <property name="text">
    <string>(Re-)initialize platform</string>
   </property>
  </action>
  <action name="actionExecuteRccFile">
   <property name="text">
    <string>Execute configuration file</string>
   </property>
  </action>
  <action name="actionConfigureBmcServer">
   <property name="text">
    <string>Configure BMC server</string>
   </property>
  </action>
  <action name="actionTrainSystemInterface">
   <property name="text">
    <string>Train system interface</string>
   </property>
  </action>
  <action name="actionStartkScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/kScreen.png</normaloff>:/resources/menuIcons/kScreen.png</iconset>
   </property>
   <property name="text">
    <string>Start konsole (One tab per booted core)</string>
   </property>
  </action>
  <action name="actionTestMemory">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/menuIcons/sknife.png</normaloff>:/resources/menuIcons/sknife.png</iconset>
   </property>
   <property name="text">
    <string>Test and init DDR3 Memories</string>
   </property>
   <property name="iconText">
    <string>Test and init DDR3 Memories</string>
   </property>
   <property name="toolTip">
    <string>Tests all DDR3 memories and intitializes them with &quot;all zero&quot;!</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionLinuxRestoreImage">
   <property name="text">
    <string>Choose sccKit default Linux image</string>
   </property>
  </action>
  <action name="actionResetPlatform">
   <property name="text">
    <string>Hardware platform reset (SCC &amp;&amp; FPGA)</string>
   </property>
  </action>
  <action name="actionCustomBmcCmd">
   <property name="text">
    <string>Custom BMC command</string>
   </property>
  </action>
  <action name="actionSetLinuxGeometry">
   <property name="text">
    <string>Set Linux virtual display resolution</string>
   </property>
  </action>
  <action name="actionEnableSoftwareRamAndUart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable software RAM and UART</string>
   </property>
  </action>
  <action name="actionPopshmSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>POP-SHM settings</string>
   </property>
  </action>
  <action name="actionCmdLineArgs">
   <property name="text">
    <string>Custom kernel command line arguments</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
